hydra-node/exe/tx-cost/Main.hs

```diff
@@ -264,20 +278,22 @@ costOfFanOut = markdownFanOutCost <$> computeFanOutCost
   markdownFanOutCost stats =
     unlines $
       [ "## Cost of FanOut Transaction"
-      , " Involves spending head output and burning head tokens. Uses ada-only UTxO for better comparability."
+      , "Involves spending head output and burning head tokens. Uses ada-only UTxO for better comparability."
       , ""
-      , "| UTxO  | Tx size | % max Mem | % max CPU |"
-      , "| :---- | ------: | --------: | --------: |"
+      , "| UTxO  | Tx size | % max Mem | % max CPU | Min fee ₳ |"
+      , "| :---- | ------: | --------: | --------: | --------: |"
       ]
         <> fmap
-          ( \(numElems, txSize, mem, cpu) ->
+          ( \(numElems, txSize, mem, cpu, Lovelace minFee) ->
```

Comment:
what if we remove the code duplicate we have?
I think would be much better and less error-prone.